src/main/java/com/datasonnet/document/JavaObjectDocument.java

```diff
@@ -0,0 +1,24 @@
+package com.datasonnet.document;
+
+public class JavaObjectDocument implements Document<Object> {
+
+    private Object contents;
+
+    public JavaObjectDocument(Object contents) {
+        this.contents = contents;
+    }
+
+    @Override
+    public Object getContents() {
+        return contents;
+    }
+
+    @Override
+    public String getMimeType() {
+        return "application/java";
```

Comment:
DW uses `application/java`. I also saw `application/java-serialized-object`, `application/java-byte-code`, `application/x-java-class`, etc. Oddly enough, https://www.iana.org/assignments/media-types/media-types.xhtml does not mention anything.
I think we should keep just `application/java` for the sake of brevity and simplicity.

Reply:
Technically java objects can't have a mime type, because mime types are for byte streams only and in memory programmatic data structures don't count, but in practice there're a few. For all that Mule uses that, by far the biggest player that has a mime type for java objects is Spring, which is who uses "application/x-java-class", which is also at least a _legal_ mime type because of the `x`, making it an extension. `application/java` is a mimetype that would only be legal if registered. The plugin should accept them all, of course.